lib/ddtrace/contrib/action_cable/instrumentation.rb

```diff
@@ -0,0 +1,34 @@
+module Datadog
+  module Contrib
+    module ActionCable
+      module Instrumentation
+        # Instruments ActionCable::Connection to ensure the top-level
+        # rack request has the correct resource for this WebSockets connection.
+        #
+        # Some of our common assumptions don't hold under WebSockets, one example
+        # being the http status code, which is -1 for this request.
+        module ActionCableConnection
+          def on_open
+            Datadog.tracer.trace('action_cable.on_open') do |span|
```

Comment:
This span name should be moved into `Ext` as a constant.

Reply:
✅